requirements.txt

```diff
@@ -44,7 +44,6 @@ gitdb2==2.0.0
 GitPython==2.1.0
 glob2==0.5
 gunicorn==19.6.0
-# Using patched honcho that includes: https://github.com/nickstenning/honcho/pull/179
```

Comment:
@itsjeyd  
1. I've update the test instructions to include (uninstalling flask and archiving an appserver)
2. I'll add the comment back to the requirements.txt.
3. I've rebased master.
4. I've actually followed the instructions in the Handbook with regards to PRs and squashed my commits into one. Looking at my reflog, it will be quite difficult to undo this unfortunately. Hope this is ok with you.

Reply:
@DevanR Thanks for the updates!

> I've update the test instructions to include (uninstalling flask and archiving an appserver)

I made a small follow-up change to the PR description: What we need to do as part of testing these changes is archive an instance; app servers don't have functionality for archiving them :)

> I've actually followed the instructions in the Handbook with regards to PRs and squashed my commits into one. Looking at my reflog, it will be quite difficult to undo this unfortunately. Hope this is ok with you.

One commit is fine! I'd suggest adjusting the commit message a little bit, though, since it has some info that's no longer relevant. You could go from this:

```
Update Ocim Ansible package

Removed Flask==0.11.1

Add missing PyYAML dependency in cleanup_utils/requirements.txt

Upgrading ansible from 2.1.2.0 to 2.7.9 for collect_activity and load_balancer_conf

Added comment back to requirements.txt
```

... to this:

```
Update dependencies with security vulnerabilities:

- Remove Flask==0.11.1 dependency.
- Add missing PyYAML dependency in cleanup_utils/requirements.txt.
- Upgrade ansible to 2.7.9 for collect_activity.yml and load_balancer_conf.yml playbooks.
```